tests/shallowMount.spec.ts

```diff
@@ -91,6 +91,28 @@ describe('shallowMount', () => {
     )
   })
 
+  it('stubs all components, but allows disabling stub by passing false', () => {
+    const wrapper = mount(ComponentWithChildren, {
+      shallow: true,
+      global: {
+        stubs: {
+          Hello: false
+        }
+      }
+    })
+    expect(wrapper.html()).toEqual(
+      '<div class="ComponentWithChildren">\n' +
```

Comment:
I wonder if we can use the `inlineSnapshot` feature from Jest here? Or do you think the string assertion is easier to read?

Thanks for all these PRs - let's do a release as soon as they are all merged up.

Reply:
I prefer to keep this as it is for now - just for consistency with style of other tests here. We could probably address it in separate clean-up MR

Reply:
A bit offtopic, regarding release. I believe I have some more (~5-10 MR coming) somewhere in next two days. I'm working (for fun) on building a prototype migration of `gitlab-ui` to Vue3 using compat build and it is surprisingly satisfying. 

With new plugin system in play I was able to mimic a huge precent of old VTU v1 behavior in pretty clean manner (for now we would like to have tests which run on both versions of VTU and)

Reply:
Ok, happy to keep this as-is.

If you have 5-10 more PRs coming, we can wait until they are all merged up before doing a release.

Reply:
@lmiller1990 Sure, I'll drop a comment in "last PR" for letting you know. thank you for acting super-fast